src/web/src/components/TelescopeAvatar.tsx

```diff
@@ -1,5 +1,14 @@
 import { MouseEventHandler } from 'react';
 import { Avatar } from '@material-ui/core';
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    image: {
+      border: `1px solid ${theme.palette.border.main}`,
```

Comment:
I notice GitHub uses `0 0 0 1px rgba(27,31,36,0.15)` for light mode and `0 0 0 1px rgba(240,246,252,0.1)` for dark mode.
Since we're using their avatar let's copy them as is.

Reply:
Actually i converted rgba(...) to hex code